TestDanger/AppDelegate.swift

```diff
@@ -16,6 +16,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        #warning("add warning")
+
+
+
+
+        let tutu = "qewr"
+
+        var test = 123
+
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/HugoSay/TestDanger/blob/dd8912e7a9894b158abcc3820e7cc1094702951d/TestDanger/AppDelegate.swift#L28"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>